.github/workflows/cmake.yml

```
@@ -265,6 +265,14 @@ jobs:
         pattern: '^\S.*?: .+'
         error: 'Missing category in commit title (if this is a fix up of a previous commit, it should be squashed)'
 
+    - name: Check title
+      if: ${{ success() || failure() }}
+      uses: tim-actions/commit-message-checker-with-regex@v0.3.1
+      with:
+        commits: ${{ steps.get-pr-commits.outputs.commits }}
+        pattern: '^.+?[^\.\n](\n.*?)*$'
```

```suggestion
        pattern: '^.+[^.\n](?:\n.*)*$'
```

- No need to escape `.` in character set
- We don't care about the capturing group
- No need not to be greedy in either cases, I think?

---

:+1: on points 1 and 3, but we only get a boolean return of matched or not, so it doesnt matter if they capture or not :)